### ISSUE TITLE
Storybook

### ISSUE THREAD

**Issue by abernier** [pmndrs/postprocessing#465]
Would add some confort if, like it was recently [added in `drei-vanilla`](https://github.com/pmndrs/drei-vanilla/pull/1), we could have Storybook here also.

**Comment by vanruesc** [comment]
There's already a [manual](https://github.com/pmndrs/postprocessing/tree/v7/manual) in the works on the v7 branch that is intended to serve as an in-depth guide for `postprocessing`. Does that cover this or is Storybook something else?

**Comment by LucaArgentieri** [comment]
Yes, the purpose is the same, that is to offer an organized list of effects, as the examples of threejs

Maybe [storybook](https://storybook.js.org/) brings some extra functionality like test and show snippets of code if I’m not mistaken, plus obviously the various states and props of the component/scene.

**Comment by vanruesc** [comment]
Yeah, I also think that the manual will suffice, so I'm closing this for now.

We can revisit the idea of adding Storybook later when the manual is live. Maybe we can add a Storybook section or something along those lines at some point.
